crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -91,8 +91,8 @@
 
     private static final Path OBJECT_TYPE_PATH = new Path("objectType");
 
-    private final Metadata metadata;
-    private final Factory factory;
+    public final Metadata metadata;
+    public final Factory factory;
```

Comment:
Used in getSavedSearch. No need to hide these as they're final, either add getters, or make public. This seems more straight forward. Without exposing these, I have to go into configuration to get an instance of Metadata, and that's introducing cyclic dependency,